functions/APIs/queries.js

```diff
@@ -1,7 +1,35 @@
 const { db } = require('../util/admin');
 const axios = require('axios');
+const _ = require('lodash');
 require('dotenv').config();
 
+//calculate distances between farms and food banks
+function calculateDistances(data, request) {
+    return new Promise(function(resolve, reject) {
```

Comment:
All of the anonymous functions in this file can be arrow functions.  The only time you want the `function` keyword (to a reasonable approximation) is when the function is a top-level member of your module.